pandas/io/clipboard.py

```diff
@@ -43,12 +44,12 @@ def read_clipboard(**kwargs):  # pragma: no cover
 
     counts = set([x.lstrip().count('\t') for x in lines])
     if len(lines) > 1 and len(counts) == 1 and counts.pop() != 0:
-        kwargs['sep'] = '\t'
+        sep = '\t'
 
-    if kwargs.get('sep') is None and kwargs.get('delim_whitespace') is None:
-        kwargs['sep'] = '\s+'
+    if sep is None and delim_whitespace is None:
+        sep = '\s+'
 
-    return read_table(StringIO(text), **kwargs)
+    return read_table(StringIO(text), sep, delim_whitespace, **kwargs)
```

Comment:
Could you make these keyword arguments (`sep=sep, delim_whitespace=delim_whitespace`)